cypress.config.js

```diff
@@ -10,13 +10,17 @@ module.exports = defineConfig({
   requestTimeout: 60000,
   responseTimeout: 60000,
   e2e: {
-    // this config comes dusring v12 update
+    // This config comes dusring v12 update.
```

Comment:
if you work on that comment please also correct "dusring" :)

Reply:
fixed typo! °_°